Source/Core/Request.js

```diff
@@ -0,0 +1,80 @@
+/*global define*/
+define([
+        './defaultValue'
+    ], function(
+        defaultValue) {
+    "use strict";
+
+    /**
+     * Stores information for making a request using {@link RequestScheduler}.
+     *
+     * @exports Request
+     *
+     * @private
+     */
+    function Request(options) {
+        options = defaultValue(options, defaultValue.EMPTY_OBJECT);
+
+        /**
+         * The URL to request.
+         *
+         * @private
```

Comment:
Since the class is private, each member does not need to be `private`.  I would just mark ones private if they are only intended to be used by `RequestScheduler` (looks like `startPromise` and `server`).